### PR TITLE
build: Remove autotools build path

### PR DESCRIPTION
**Pull request by Joshua-Ashton**

### CODE REVIEW

**Review by HansKristian-Work — APPROVED** [review]
LGTM, @doitsujin should also approve as it touches the meta stuff.

File: libs/vkd3d/vkd3d_shaders.h [hunk 1]
```diff
@@ -19,778 +19,28 @@
...
-    0x00000049,0x00000048,0x00000000,0x0003003e,0x00000029,0x00000049,0x000200f9,0x00000040,
-    0x000200f8,0x00000040,0x0004003d,0x00000012,0x0000004d,0x00000029,0x0003003e,0x0000004c,
-    0x0000004d,0x0003003e,0x0000004b,0x0000004d,0x000100fd,0x00010038
-};
+typedef enum VKD3D_META_COPY_MODE
+{
+    VKD3D_META_COPY_MODE_1D = 0,
+    VKD3D_META_COPY_MODE_2D = 1,
+    VKD3D_META_COPY_MODE_MS = 2,
+} VKD3D_META_COPY_MODE;
```
Comment (doitsujin) [inline libs/vkd3d/vkd3d_shaders.h:27]: now I don't mind making this an enum, but it should be consistent with the rest of the code, i.e.
```
enum vkd3d_meta_copy_mode
{
...
```
with no typedef.

**Review by doitsujin — CHANGES_REQUESTED** [review]

File: libs/vkd3d/vkd3d_shaders.h (hunk 1 above)
Comment (Joshua-Ashton) [reply libs/vkd3d/vkd3d_shaders.h:27]: Can do.

Comment (Joshua-Ashton) [reply libs/vkd3d/vkd3d_shaders.h]: Resolved.

**Pull request merged**